Code/Commando/natsock.cpp

```diff
@@ -245,7 +245,7 @@ bool SocketHandlerClass::Open(int inport, int outport)
 	/*
 	** Set the blocking mode of the socket to non-blocking.
 	*/
-	unsigned long nonblocking = true;
+	u_long nonblocking = true;
```

Comment:
is this intentional?

Reply:
ioctlsockets takes `u_long *` as its 3rd parameter.